monitoring/prometheus/prometheus.yml

```diff
@@ -1,6 +1,6 @@
 # This config is required for Prometheus to scrape other containers
 scrape_configs:
-- job_name: court-backend
+- job_name: default/court-backend
```

Comment:
just curious, but why adding `default` here?

Reply:
When scraping in k8s all resources have a namespace, which is "default" by default :)
This makes the dashboard parsing the same on localhost and in k8s